src/modules/account/actions/approve-legacy-rep.js

```diff
@@ -6,14 +6,15 @@ import { updateAssets } from 'modules/auth/actions/update-assets'
 
 export default function (universeID, callback = logError) {
```

Comment:
Bridging from convo over in discord — I’d intentionally left these out since they were original approved w/out.  My thinking was that we had left them off since they’re only for testnet purposes, but I’ll ask to clarify next time!  

Do we want coverage on these?  Happy to add.

Reply:
Excluding these tests per our discussion.